packages/@o3r/schematics/src/rule-factories/dev-tools/devtools-registration.ts

```diff
@@ -52,6 +53,64 @@ const registerModule = (options: DevtoolRegisterOptions): Rule => (tree, context
   return tree;
 };
 
+/**
+ * Rule to inject a service after `bootstrapModule`
+ * @param options
+ */
+export const injectServiceInMain = (options: DevtoolRegisterOptions): Rule => (tree, context) => {
+  if (!options.serviceName) {
+    return tree;
+  }
+
+  const mainFilePath = getMainFilePath(tree, context, options.projectName);
+
+  if (!mainFilePath || !tree.exists(mainFilePath)) {
+    return tree;
+  }
+
+  const content = tree.readText(mainFilePath);
+  if (content.includes(`inject(${options.serviceName})`)) {
+    return tree;
+  }
+
+  const match = /platformBrowserDynamic\(\).bootstrapModule\(\w+\)/.exec(content);
```

Comment:
we should probably support standalone apps as well, since it's now the default with Angular 17+